project4/index.html

```diff
@@ -0,0 +1,197 @@
+                <p>APDC2019 is an annual design conference held to explore and educate designers about the role,
+                    practices, and most important evolution of political design. The subjects range from the graphic
+                    design of the campaigns to the creative tactics used in message and positioning of candidates,
+                    propositions, and referendums.
+                </p>
+            </section>
+
+            <!-- mission -->
+            <h3 id="mission">Mission</h3>
+            <section class="left_border">
+                <p>Since its inception, the mission of the APDC is to educate designers on how to reach people
+                    regardless of color, creed, or political ideology. With that in mind, any one in attendance of APDC
+                    must respect the political ideology of any other attendees, no matter their differences.Leave your
+                    party
+                    at the door and remember: We are all here to educate and connect on the basis of accurate and
+                    informative
+                    communications.</p>
+                <img src="images\asset4.png"
+                    alt="a group of protesters marching while holding sign that generally seem to be anti-hate speech">
+            </section>
+
+            <!-- speakers -->
+            <h3 id="speakers">Keynote Speakers</h3>
+            <section class="right_border">
+                <div class="speaker_wrapper">
+                    <div class="speaker_card">
+                        <img class="speaker-image" src="images\asset3.png"
+                            alt="a photo of scott thomas speaking at a podium">
+                        <h4>Scott Thomas</h4>
+                        <p>As the Design Director for Obama’s 2008 campaign, Scott Thomas led a now-historic political
+                            campaign, in which branding, design, and the web played a truly pivotal role. Likening the
+                            experience to “building an airplane in flight,” Scott talks about the creative’s need for
+                            triage, the crucial role of incremental design improvements, and the importance of getting
+                            back to the hand and keeping things simple.
+                        </p>
+                        <h4 class="talk_title">Talk: Making Obama</h4>
+                        <p class="talk_details">How did an obscure senator rise to the highest office in the land and
+                            celebrity status? With the aid of branding and design. Scott details his involvement in
+                            Obama's campaign on Day 1.
+                        </p>
+                        <!-- https://99u.adobe.com/videos/5821/scott-thomas-designing-the-obama-campaign -->
+                    </div>
+                    <div class="speaker_card">
+                        <img class="speaker-image" src="images\asset2.png"
```

Comment:
```suggestion
                        <img class="speaker-image" src="images/asset2.png"
```
~Okay. I ran out of Slash GIFs.~ I lied.
![the other slash](https://media.giphy.com/media/ytlPLqdKk0Df2/giphy.gif)